### ISSUE TITLE
Allow user to upload PDF document(s)

### ISSUE THREAD

**Issue by kuoirene** [amosproj/amos2024ss05-knowledge-graph-extractor#20]
## User story
1. As a user
2. I need to upload PDF documents
3. So that the PDF can later be parsed and turned into a knowledge graph

## Acceptance criteria
* There should be a user interface where the user can drag and drop or select PDF files for upload
* The user has an upload button to trigger the upload
* There should be a rest endpoint in fastAPI to receive form data from the upload
* Files uploaded need to be of type PDF, return error message to user stating wrong format and upload cannot be done
* CORS implementation done separately: https://github.com/amosproj/amos2024ss05-knowledge-graph-extractor/issues/62, everything except this connection should work.

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
